rainbowstream/rainbow.py

```diff
@@ -406,13 +407,54 @@ def search():
         printNicely(magenta('I\'m afraid there is no result'))
 
 
+def upload_twitter_media(image_paths):
+    """
+        Upload media and images to wtiter using their api
+    """
+    image_ids = []
+    for image_path in image_paths:
+        image_path = image_path.strip()
+        image_data = open(image_path, 'rb').read()
```

Comment:
Please consider to use `os.path.expanduser(image_path)` instead of just `image_path` to enable opening links as `~/Desktop/test.png`